src/stylesheets/timeline-comment.scss

```diff
@@ -84,11 +84,18 @@
     border-top: $border;
 
     &[reaction-count="0"] {
-      display: none;
+      height: 0;
+      overflow: hidden;
+      opacity: 0;
     }
 
     &:not(:hover) .reactions-popover:not([open]) {
-      display: none;
```

Comment:
I like the effect but opacity:0 leaves the element focusable via keyboard navigation.

Reply:
@jdanyow Good catch.  I have added a `tabindex="-1"` attribute so this won't happen.  This preserves the existing behavior of simply not being able to select the add reactions button using keyboard navigation unless the mouse is hovered over the button (this is both what utterances and GitHub currently do).